tools/taskcluster-proxy/build.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
+
+cd "$(dirname "${0}")"
+
+# Support go 1 release 1.9 or higher
+GO_MAJOR_VERSION=1
+MIN_GO_MINOR_VERSION=9
```

Comment:
I think the comment (line 5) and `MIN_GO_MINOR_VERSION` (line 7) should probably be updated to require go1.13.